package.json

```diff
@@ -1,11 +1,14 @@
 {
-  "name": "fence",
-  "version": "0.2.11",
+  "name": "@guardian/fence",
+  "version": "0.2.13",
   "description": "Utility to render fenced embeds in iframes",
   "main": "fence.js",
   "directories": {
     "examples": "examples"
   },
+  "files": [
+    "Gruntfile.js"
```

Comment:
Should this be `fence.js`?

Reply:
[`npm pack`](https://docs.npmjs.com/cli/v7/commands/npm-pack) is a useful command to see the content that would be published.

Reply:
`fence.js` is automatically included, since it is defined in [`main`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#:~:text=Certain%20files%20are,the%20%22main%22%20field). I wasn't sure if `Gruntfile.js` is needed, but included it just to be safe.

The package only has these two files now, other than the defaults.
<img width="139" alt="Screenshot 2022-04-20 at 11 25 53" src="https://user-images.githubusercontent.com/79363218/164211019-0c61c849-6179-4a08-b9cc-8f60438edf0e.png">

Reply:
> `fence.js` is automatically included, since it is defined in [`main`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#:~:text=Certain%20files%20are,the%20%22main%22%20field).

Ah! TIL!

Grunt is a task runner, typically only used during development, so I wouldn't think it's needed. Not a blocker though.

Reply:
The [readme mentions](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#:~:text=Certain%20files%20are,the%20%22main%22%20field) using Grunt to test the installation, so felt it could be useful to include.